src/App.test.js

```diff
@@ -192,6 +192,18 @@ describe('error message displaying at correct times', () => {
 
     expect(getByTestId("result")).toHaveTextContent("21")
   });
+
+  test("returns 0 instead of an empty string", () => {
```

Comment:
This looks unrelated to the bug.

Reply:
I had forgotten to change the wording for what this test is testing.

Reply:
Fixed the wording.

Reply:
Are the changes pushed?

Reply:
Yes